windows/windows_update/win_update_restore_snapshot.yml

```diff
@@ -0,0 +1,40 @@
+# Copyright 2023 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+# Description:
+#   This case is to restore VM to the original base snapshot, which is not installed with the msu file
```

Comment:
Is this also a new test case?

Reply:
What's the meaning of this playbook? If it is just for restore old base snapshot before a new test, we have defined below global vars to clean and restore snapshot. 
```
# cleanup_old_snapshots: false
# revert_to_snapshot: 'snapshot_name'
```

Reply:
Thanks, will try the recommended parameter instead of adding a new test case.